lib/kuzzle/event/KuzzleEventEmitter.ts

```diff
@@ -360,8 +403,8 @@ async function pipeCallback(error, ...updated) {
     await Bluebird.map(corePipes, (fn) => fn(...updated));
   }
 
-  for (let i = 0; i < this.events.length; i++) {
```

Comment:
This old for loop was choose to increase performance, I don't know if `for (let i = 0; i < ...)` is still faster than `for (const ... of ...)` in later Node.js versions